.gitignore

```diff
@@ -29,4 +29,8 @@ bld/
 */dist/
 node_modules
 .vscode
-*.vsix
\ No newline at end of file
+*.vsix
+*.map
+*.js
+NewmanPostman/npm-debug.log
+.taskkey
```

Comment:
Similar to .js above. If we don't need the .taskkey file, then we should remove it from source as well.

Reply:
Done, please consider PR#49